docs/data/material/migration/migrating-from-deprecated-apis/migrating-from-deprecated-apis.md

```diff
@@ -30,22 +30,22 @@ If you need to run a specific codemod, those are also linked below.
 
 ### TransitionComponent
 
-The Accordion's `TransitionComponent` was deprecated in favor of `slots.transition` ([Codemod](https://github.com/mui/material-ui/tree/master/packages/mui-codemod#accordion-props)):
+The Accordion's `TransitionComponent` was deprecated in favor of `slots.transition` ([Codemod](https://github.com/mui/material-ui/tree/HEAD/packages/mui-codemod#accordion-props)):
```

Comment:
The only potential opportunity I could spot:

```suggestion
The Accordion's `TransitionComponent` was deprecated in favor of `slots.transition` ([Codemod](https://github.com/mui/material-ui/tree/HEAD/packages/mui-codemod#accordion-props)).
We are making this change to normalize the customization of nested elements with a single `slots` prop:
```

Reply:
I agree. I'll add a single explanation for the overarching slot pattern standardization on top of the guide and link it to each deprecation. I want to release a brief blog about it alongside it so we can share it. I'll handle this one separately this week. (Also replied here https://github.com/mui/material-ui/pull/40767#discussion_r1481475704)